monai/config/deviceconfig.py

```diff
@@ -200,7 +200,7 @@ def get_gpu_info() -> OrderedDict:
 
     if num_gpus > 0:
         _dict_append(output, "Current device", torch.cuda.current_device)
-        if hasattr(torch.cuda, "get_arch_list"):  # get_arch_list is new in torch 1.7.1
+        if hasattr(torch.cuda, "get_arch_list"):  # `get_arch_list` is new in torch 1.7.1
```

Comment:
how do you test this, it seems `torch.cuda.get_arch_list` is in v1.6.0 as well:
https://github.com/pytorch/pytorch/blob/v1.6.0/torch/cuda/__init__.py#L363

Reply:
Hi @wyli ,

Thanks for your review and comments.
Only this API was already added version info by someone in the comment for this API so I didn't check it...
Let me try to update it ASAP.

Thanks.

Reply:
Updated in the latest commit and double checked all the items in https://pytorch.org/docs/stable/index.html.

Thanks.